### PR TITLE
audit log data via REST API.

### PR DESCRIPTION
**Pull request by alwell-kevin**
Adding more clarity on how to retrieve git audit log data via REST API.

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
@alwell-kevin Thanks so much for opening a PR! I'll get this triaged for review ✨

File: content/github/setting-up-and-managing-organizations-and-teams/reviewing-the-audit-log-for-your-organization.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ To ensure a secure IP and maintain compliance for your organization, you can use
 {% data reusables.audit_log.audit-log-api-info %}
 
 {% if currentVersion == "free-pro-team@latest" %}
-Note that you can't retrieve Git events using the GraphQL API.
+Note that you can't retrieve Git events using the GraphQL API. Instead, try the [REST API](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/reviewing-the-audit-log-for-your-organization#git-category-actions).
```
Comment (shati-patel) [inline content/github/setting-up-and-managing-organizations-and-teams/reviewing-the-audit-log-for-your-organization.md:130]:
```suggestion
Note that you can't retrieve Git events using the GraphQL API. To retrieve Git events, use the REST API instead. For more information, see "[`git` category actions](git-category-actions)." 
```

**Review by shati-patel — COMMENTED** [review]
Thank you for the update @alwell-kevin! 

One of the tests is failing because we don't tend to use hard-coded links within our documentation. I'll add a suggestion that should fix it 😊

File: content/github/setting-up-and-managing-organizations-and-teams/reviewing-the-audit-log-for-your-organization.md [hunk 2]
```diff
@@ -127,7 +127,7 @@ To ensure a secure IP and maintain compliance for your organization, you can use
 {% data reusables.audit_log.audit-log-api-info %}
 
 {% if currentVersion == "free-pro-team@latest" %}
-Note that you can't retrieve Git events using the GraphQL API.
+Note that you can't retrieve Git events using the GraphQL API. To retrieve Git events, use the REST API instead. For more information, see "[`git` category actions](git-category-actions)."
```
Comment (shati-patel) [inline content/github/setting-up-and-managing-organizations-and-teams/reviewing-the-audit-log-for-your-organization.md:130]:
```suggestion
Note that you can't retrieve Git events using the GraphQL API. To retrieve Git events, use the REST API instead. For more information, see "[`git` category actions](#git-category-actions)." 
```

Oops, forgot the `#` for the anchor 🤦🏽

**Review by shati-patel — APPROVED** [review]
I'll get this merged shortly. Thanks again for the PR 😃

**Pull request merged**